cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -54,7 +54,13 @@ def __init__(self,
         self.v = np.zeros(self.n, dtype=bool)
         self.s = np.zeros(self.n, dtype=bool)
 
-        self.omega = 1
+        # Keep initial state positive by transferring the negative part to the
+        # global phase.
+        if initial_state < 0:
```

Comment:
When does negative state occur?

Reply:
Since StabilizerStateCHForm keeps track of the global phase, a negative initial state should just be the positive one with a -1 phase. Currently, it was using the negative number bit representation which is strictly worse IMO.

Reply:
Got it - as discussed let's make this into an error case instead - let's force computational basis state initial states.

Reply:
Will make that change in a separate PR since initial_state is an arg for multiple classes. Would prefer to do put them all in a single PR.